PROJECT

```diff
@@ -10,6 +10,9 @@ resources:
 - group: mattermost.com
   kind: ClusterInstallation
   version: v1alpha1
+- group: mattermost
```

Comment:
I wonder if we could use some different group name as the full resource name will be: `mattermost.mattermost.mattermost.com` 😄 
Maybe something like `cloud` would make sense? I do not really have any good idea here.

Reply:
Yeah, I struggled with this as well, but also struggled to figure out what the better option would be. `cloud` sorta makes sense to me, but not sure if something else would make more sense. Open to suggestions on this as well.

Reply:
Oof that's gross :P Maybe `platform`?